templates/docs/master_doc.rst_t

```diff
@@ -0,0 +1,21 @@
+.. {{ project }} documentation master file, created by
+   sphinx-quickstart on {{ now }}.
```

Comment:
The "created by sphinx-quickstart" is a bit annoying, and is an example of how `copy` targets might be modified , here after copy, replace "sphinx-quickstart" with `moban` or `yehua`

`{{ now }}` doesnt exist yet, and is a bad idea for re-generated files.

maybe we can delete these lines by building a slightly different 'include' command.